AllJoyn/Platform/Bridge/BridgeNET/BridgeNET.vcxproj

```diff
@@ -211,7 +212,9 @@
     <ClInclude Include="Stdafx.h" />
   </ItemGroup>
   <ItemGroup>
-    <ClCompile Include="AssemblyInfo.cpp" />
+    <ClCompile Include="AssemblyInfo.cpp">
+      <RuntimeLibrary Condition="'$(Configuration)|$(Platform)'=='Release|x64'">MultiThreadedDLL</RuntimeLibrary>
```

Comment:
Not sure why this is generated this way. But I see that it is /MD for all other configurations as well. So same for all configs & other files too.

---
In reply to: [75553518](https://github.com/tjaffri/msiot-samples/pull/9#discussion_r75553518) [](ancestors = 75553518,75537238,75174478)